xbmc/platform/darwin/ios-common/DarwinNSUserDefaults.h

```diff
@@ -1,28 +1,50 @@
+#pragma once
 /*
  *  Copyright (C) 2015 Team MrMC
  *      https://github.com/MrMC
  *
- *  SPDX-License-Identifier: GPL-2.0-or-later
- *  See LICENSES/README.md for more information.
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with MrMC; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
```

Comment:
Why?

Reply:
Was a backport from mrmc codebase, copied the licensing as is from that

Reply:
Please use SPDX licensing header.